docs/INSTALL.md

```diff
@@ -31,6 +32,37 @@ methods:
 curl -fsSL https://d2lang.com/install.sh | sh -s -- --help
 ```
 
+### Security
+
+The install script is not the most secure way to install d2. We recommend that if
+possible, you use your OS's package manager directly or install from source with `go` as
+described below.
+
+But this does not mean the install script is insecure. There is no major flaw that
+the install script is more vulnerable to than any other method of manual installation.
+The most secure installation method involves a second independent entity, i.e your OS
+package repos or Go's proxy server.
+
+We're careful shell programmers and are aware of the many footguns of the unix shell. Our
+script was written carefully and with detail. For example, it is not vulnerable to partial
+execution and the entire script runs with `set -eu` and very meticulous quoting.
+
+It follows the XDG standards, installs d2 properly into a unix hierarchy path (defaulting
```

Comment:
D2

Reply:
No here we should stick to lowercase as I'm referring to the `d2` command not D2. I'll add backquotes.